android/src/BluetoothSensor.java

```diff
@@ -138,13 +138,13 @@ private void engineSensorDataToListeners(BluetoothGattCharacteristic c) {
     if(pressure != 0)
       listener.onBarometricPressureSensor(pressure / 100.0f, 0.01f);
 
-    final int ignitions_per_sec = c.getIntValue(c.FORMAT_UINT16, 11);
+    final int ignitions_per_second = c.getIntValue(c.FORMAT_UINT16, 11);
```

Comment:
These changes confuse me **every time** I re-read an amended version of your PR. Doesn't belong here.

Reply:
If I put this in an own commit, can you merge it? Its just renaming of a variable...